recipes-bsp/tegra-binaries/tegra-helper-scripts/tegra-flash-helper.sh

```diff
@@ -24,24 +24,10 @@ blocksize=4096
 # need to do anything when run from this script, as we
 # have already copied needed files to the current working
 # directory.
-mkfilesoft() {
-    :
-}
-
-cp2local() {
-    :
-}
-
-signimage() {
-    :
-}
-
-process_l4t_conf_dtbo() {
-    :
-}
-
 partition_exists_in_PT_table() {
-    [ "$1" = "BCT-boot-chain_backup" ]
+    local partname="$1"
+    local layoutfile="$2"
```

Comment:
while we demand /bin/bash as shell, I would suggest to avoid bashism when possible, here we can avoid using `local` keyword and pass $1 and $2 directly to script below

Reply:
Unfortunately I long ago had to give up on unbashifying the script, since I couldn't easily come up with ways of translating some of the bashisms I had to borrow from NVIDIA's script into POSIX shell that would produce the exact same results.